aws/resource_aws_fsx_lustre_file_system.go

```diff
@@ -137,6 +133,12 @@ func resourceAwsFsxLustreFileSystem() *schema.Resource {
 				Computed:     true,
 				ValidateFunc: validation.IntBetween(0, 35),
 			},
+			"auto_import_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
Should this default to `NONE` to [match the API](https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystemLustreConfiguration.html#FSx-Type-CreateFileSystemLustreConfiguration-AutoImportPolicy) and enable drift detection when it is unconfigured?

Reply:
I tried to play to play with this but want able to find a combination other than the one in the PR to maked this work.